app/code/BVMyBlog/Blog/Controller/Adminhtml/Post/Delete.php

```diff
@@ -0,0 +1,62 @@
+<?php
+declare(strict_types=1);
+namespace BVMyBlog\Blog\Controller\Adminhtml\Post;
+
+use Magento\Framework\App\Action\HttpGetActionInterface;
+
+/**
+ * Class Delete
+ *
+ * Controller Delete
+ */
+class Delete extends \Magento\Backend\App\Action implements HttpGetActionInterface
+{
+    protected $resultPageFactory;
+    protected $contactFactory;
+
+    /**
+     * Construct
+     *
+     * @param \Magento\Backend\App\Action\Context $context
+     * @param \Magento\Framework\View\Result\PageFactory $resultPageFactory
+     * @param \BVMyBlog\Blog\Model\PostFactory $contactFactory
+     * @return string
+     */
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        \Magento\Framework\View\Result\PageFactory $resultPageFactory,
+        \BVMyBlog\Blog\Model\PostFactory $contactFactory
+    ) {
+        $this->resultPageFactory = $resultPageFactory;
+        $this->contactFactory = $contactFactory;
+        parent::__construct($context);
+    }
+
+    /**
+     * Function execute
+     */
+    public function execute()
+    {
+        $id = $this->getRequest()->getParam('id');
+
+        $contact = $this->contactFactory->create()->load($id);
+
+        if (!$contact) {
+            $this->messageManager->addError(__('Unable to process. please, try again.'));
```

Comment:
If there is no blog post with such **ID** we should notify a user about it. Please change message